src/main/java/io/lighty/yang/validator/Main.java

```diff
@@ -172,18 +173,23 @@ private static void runLyvForProvidedFolder(final Configuration config, final Fo
         }
     }
 
+    @SuppressFBWarnings(value = "SLF4J_SIGN_ONLY_FORMAT",
```

Comment:
No, this is not true - we are not using logback to create output.

Reply:
Removed